### ISSUE TITLE
Release ros-jade-urdf

### ISSUE THREAD

**Issue by scpeters** [ros/robot_model#102]
I'll talk to @tfoote to see if I can help.

**Comment by scpeters** [comment]
Also `urdf_parser_plugin` needs to be released into jade.

**Comment by scpeters** [comment]
Waiting on https://github.com/ros-planning/geometric_shapes/issues/27

**Comment by scpeters** [comment]
[geometric_shapes has been released](https://github.com/ros/rosdistro/pull/8080), so I think this is not blocked by anything anymore.

**Comment by wjwwood** [comment]
I'm working on it :smile:

**Comment by wjwwood** [comment]
@isucan I made two commits to the repository (other than the one that removed `resource_retriever`) and then ran a prerelease for Indigo. The prerelease ended up failing on one of the dependencies, but that looks unrelated to `robot_model`. I looked through the logs of the prerelease and I think it tested enough of it to say that it's safe to release.

I also manually reviewed the changes staged for the next indigo release and it looks ok to me.

So, I'm going to release for Indigo and then the same version into Jade. FYI
